Add message to SolidusShipStation::Api::RequestError (#25)
solidusio-contrib/solidus_shipstation: CHANGELOG.md

# Changelog

## v2.0.0

**Breaking changes**

> - The default message for a `RequestError` has been changed from the response body of the response instead include a parsed version of this body.  If your code depends on the response body you need to access it through the `.response_body` attribute on the error instead.

Reviewer: I think that the wording is a bit confusing. Something like the following could be clearer:

```suggestion
The default message for a `RequestError` has been changed from the raw response body to a parsed version. If your code depends on the response body, you need to access it through the `.response_body` attribute on the exception instead.
```
